projects/defilyio/index.js

```diff
@@ -0,0 +1,10 @@
+const utils = require("../helper/utils");
+
+const fetch = async () => {
+  const tvl = (await utils.fetchURL("https://api.defily.io/v1/statistics")).data.payload.totalValueLocked.total;
```

Comment:
is it possible that defily is including DFL staked right into this API?

Reply:
not, I wanted to add the staking part, but I forgot it, I modified it and I'll upload the updates.